lessons/26-js-functions/index.js

```diff
@@ -0,0 +1,62 @@
+function showEvenNumbers (numMax) {
+    for (let i = 0; i <= numMax; i++) {
+        if (i % 2 !== 0) continue;
+        console.log(i);
+    }
+}
+
+showEvenNumbers(100);
+
+function showUser (name, age, phoneNumber, street, postalCode, married) {
+   
+    console.log("name: ", name,"Age: ", age,"Phone Number: ", phoneNumber,"Street: ", street, "Postal Code: ", postalCode,"Married: ", married);
+}
+
+showUser ("kevin", 29, 3063701356, "simonfraser", "s7h3t2", false);
+
+
+
+function value() {
+    for (let i = 0; i <= 100; i++) {
+        if (i % 2 !== 0) continue;
+        console.log(i);
+    }
+    
+}
+
+let evenNumber = value;
+
+evenNumber();
+evenNumber();
+evenNumber();
+evenNumber();
+evenNumber();
+
+function fibon(num) {
+    var a = 1;
+    console.log(a);
+    var b = 0;
+    console.log(b);
+    var c;
+
+    for (i = 0; i < num; i++){

```

Comment:
Also the algorithm here isn't quite right. Given:

`fibon(8)` it should produce:

```
0
1
1
2
3
5
8
13
21
34
```

Reply:
Yes you are right it would have been good if i had correct the i to c like i did in the jsbin when i finished it before. It is corrected now it will produce that result now.